engine_eval/zanzibar/authz-openfga-ex/test/prbac_mark/mark_export.json

```diff
@@ -0,0 +1,235 @@
+{
+  "id": "01GK7RA1VW5C1P19H0WEKCY0KB",
```

Comment:
we could actually delete this, just for comparison: this is an export of the actual openfga playground. didn't want to frickle around with the data model, creating structs and such, so created different files to just use json.unmarshal